### PR TITLE
All ethos text files now windows style cr/lf line endings

### PR DESCRIPTION
**Pull request by garytaylor**
All ethos text files now windows style cr/lf line endings

### CODE REVIEW

File: app/services/claim_file_builder/render_to_file.rb [hunk 1]
```diff
@@ -6,10 +6,14 @@ module RenderToFile
...
       def render_to_file(object:)
         Tempfile.new.tap do |file|
-          file.write render(object)
+          file.write with_windows_lf(render(object))
           file.rewind
         end
       end
+
+      def with_windows_lf(string)
+        string.gsub(/\n/, "\r\n")
```
Comment (leoapost) [inline app/services/claim_file_builder/render_to_file.rb]: An alternative would be to use the String [#encode](https://ruby-doc.org/core-2.3.3/String.html#method-i-encode) method.

e.g.
`string.encode(crlf_newline: true)`

File: spec/support/matchers/be_valid_et1a_claim_text.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ module Test
     class BeValidEt1aClaimTextMatcher # rubocop:disable Metrics/ClassLength
       include ::RSpec::Matchers
       def matches?(actual)
-        actual_lines = actual.lines.map { |l| l. gsub(/\n\z/, '') }
+        actual_lines = actual.lines("\r\n").map { |l| l. gsub(/\r\n\z/, '') }
```
Comment (leoapost) [inline spec/support/matchers/be_valid_et1a_claim_text.rb]: I think there's an unnecessary space char in the `l. gsub`

File: spec/support/matchers/be_valid_et1_claim_text.rb [hunk 3]
```diff
@@ -13,7 +13,7 @@ def initialize(multiple_claimants: false, representative: true)
       end
 
       def matches?(actual)
-        actual_lines = actual.lines.map { |l| l. gsub(/\n\z/, '') }
+        actual_lines = actual.lines("\r\n").map { |l| l. gsub(/\r\n\z/, '') }
```
Comment (leoapost) [inline spec/support/matchers/be_valid_et1_claim_text.rb]: I think there's an unnecessary space char in the `l. gsub`

File: app/services/claim_file_builder/render_to_file.rb (hunk 1 above)
Comment (garytaylor) [reply app/services/claim_file_builder/render_to_file.rb]: Nice one - never knew that one existed

**Pull request merged**